### PR TITLE
[28.x] Further backports and rc2

### PR DESCRIPTION
**Pull request by achow101**
* #30834
* #30807
* #30880
* https://github.com/bitcoin-core/gui/pull/835
* #30899
* #30884

### CODE REVIEW

File: src/txmempool.h [hunk 1]
```diff
@@ -23,7 +23,17 @@
 #include <util/result.h>
 #include <util/feefrac.h>
 
+// This works around a bug in Boost <= 1.80.0 when using Clang >=18.
+// See https://github.com/bitcoin/bitcoin/issues/30751.
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wenum-constexpr-conversion"
+#endif
```
[the remark this answers is not in the document]
Comment (fanquake) [reply src/txmempool.h:31]: This can now be swapped out for #30834.

[the remark this answers is not in the document]
Comment (achow101) [reply src/txmempool.h]: Backported fa9d7d5d205ada8915cbbc29599ab8e7bf1fffe0

**Review by hebasto — APPROVED** [review]
ACK 37a6739203ea3da394acef38b18a794e67543a90.

**Review by furszy — COMMENTED** [review]
utACK b97323c7aaa06bda8c76dfc754c90f9de44f4898

**Comment by hebasto** [comment]
@achow101 

Could you please add https://github.com/bitcoin-core/gui/pull/835 as well?

**Comment by achow101** [comment]
> @achow101
> 
> Could you please add [bitcoin-core/gui#835](https://github.com/bitcoin-core/gui/pull/835) as well?

Done

**Comment by hebasto** [comment]
@achow101 

Feel free to pick [this](https://github.com/hebasto/bitcoin/commit/be6354db410d162f82b5e3c76d964c0365e51d96) translation update, which resolves https://github.com/bitcoin/bitcoin/issues/30897 and includes new translations made over the nearly 3 weeks since https://github.com/bitcoin/bitcoin/pull/30715.

**Comment by fanquake** [comment]
> Feel free to pick [this](https://github.com/hebasto/bitcoin/commit/be6354db410d162f82b5e3c76d964c0365e51d96) translation update

There's not so much of a rush here that this should bypass the normal PR & review process. This should first be PR'd and merged into master and then backported. Otherwise any broken string are still in master. There is at least one other change waiting for 28.x, so rc2 also needs to either wait for that, or we'll be having an rc3 (which can include the ts update) in any case.

**Comment by hebasto** [comment]
> There's not so much of a rush here that this should bypass the normal PR & review process.

Sure.

> This should first be PR'd and merged into master and then backported.

After branching off, the Transifex.com has translation for the version branches only. Fetching 28x translations into the master branch does not look reasonable.

> Otherwise any broken string are still in master. 

Maybe this case is a special one.

**Comment by hebasto** [comment]
> > There's not so much of a rush here that this should bypass the normal PR & review process.
> 
> Sure.

Please see https://github.com/bitcoin/bitcoin/pull/30899.

**Review by stickies-v — APPROVED** [review]
ACK 06a7df70df30879e0b691d1a252636f703b8cdfb

I verified that all backports are clean, except:
- b329ed739b7311b3b47cae1ef8d576a90e0a36a1 backported from 992f83bb6f4b29b44f4eaace1d1a2c0001d43cac: merge conflict due to a difference in `sha256sum_file` import, trivially resolved

I didn't review each commit in-depth since they're clean, but the changes backported seem sensible.

**Review by hebasto — APPROVED** [review]
ACK 06a7df70df30879e0b691d1a252636f703b8cdfb, I've backported the listed PRs locally. The only merge conflict I faced was in https://github.com/bitcoin/bitcoin/pull/30807. It was trivial to resolve.

**Pull request merged**